.gitignore

```diff
@@ -8,3 +8,11 @@ hachoir-metadata/hachoir_metadata/qt/dialog_ui.py
 # generated by tox
 .tox/
 hachoir3.egg-info/
+
+# Mac files
+.DS_Store
+
+# VSCode files
+.vscode/
+
+env/
```

Comment:
This is a folder to my configs in visual studio code. Shouldn't made ir to pull request either